Makefile

```diff
@@ -1,9 +1,20 @@
+timestamp := $(shell cat latest-rulesets-timestamp)
+image := fpf.local/securedrop-https-everywhere-ruleset:$(timestamp)
+
 .PHONY: test-key
 test-key: ## Generates a test key for development/testing purposes locally.
 	openssl genrsa -out key.pem 4096
 	openssl rsa -in key.pem -outform PEM -pubout -out public.pem
 	python jwk.py > test-key.jwk
 
+.PHONY: serve
+serve: ## Builds Nginx container to serve generated files
+	@docker build --build-arg "timestamp=$(timestamp)" -t "$(image)" -f docker/Dockerfile .
+	@echo "=============================================================================="
+	@echo "          Serving ruleset at http://localhost:4080/https-everywhere/          "
+	@echo "=============================================================================="
+	@docker run --rm -p 4080 "$(image)"
```

Comment:
Specifying a single port did not work for me locally (running Docker ce 19.03.13 in Debian Stretch), had to specify the source and destination port to forward to the container. Furthermore, it might be prudent to bind only to localhost to prevent exposing containers on a local network. Based on my local testing, using the following Docker command works for me,  what do you think?

```
@docker run --rm -p 127.0.0.1:4080:4080 "$(image)"
```